Libplanet/Net/Swarm.cs

```diff
@@ -2053,59 +2053,76 @@ private void DoReply(object sender, NetMQQueueEventArgs<Message> e)
             while (!cancellationToken.IsCancellationRequested)
             {
                 _logger.Verbose("Waiting for new request...");
-                var req = await _requests.TakeAsync(cancellationToken);
-                _logger.Verbose("Request {RequestId} taken.", req.Id);
+                MessageRequest req = await _requests.TakeAsync(cancellationToken);
 
-                using (var dealer = new DealerSocket(ToNetMQAddress(req.Peer)))
+                try
+                {
+                    await ProcessRequest(req, cancellationToken);
+
+                    // Delaying dealer disposing to avoid ObjectDisposedException on NetMQPoller
+                    await Task.Delay(100, cancellationToken);
+                }
+                catch (Exception e)
```

Comment:
Some tests fail. it seems to need bypass some exceptions on some environments(i.e. mono).

Reply:
I misplaced the code that delays disposing of the dealer socket and fixed it.